src/main/java/homework_1/Main.java

```diff
@@ -1,9 +1,38 @@
 package homework_1;
+import java.util.Scanner;
 
 public class Main {
+  static final String RED = "\u001b[31m";
+  static final String RESET = "\u001b[0m";
 
   public static void main(String[] args) {
-    System.out.println("Hello homework!");
+    try {
+      dialog();
+    } catch (AlarmException e) {
+      System.out.println(e.getMessage());
+    }
+  }
+
+  public static void dialog() throws AlarmException {
```

Comment:
Please make arguments to be entered at the start of the application. No need to complicate it for now